heidelberg-university-faculty-of-medicine.csl

```diff
@@ -0,0 +1,323 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="expanded" demote-non-dropping-particle="never">
+  <info>
+    <title>Heidelberg University - Faculty of Medicine</title>
```

Comment:
The VetMed Austria style is the one that got me into CSL in 2011 and then reintroduced me couple of months ago to fix some outstanding mistakes. ;)